packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -85,6 +85,7 @@ export class MyAlgoWalletSession {
 			}
 		} catch (err) {
 			error(err);
+			throw err
```

Comment:
What does `error(err)` does then?

Reply:
it logs the error (`error` is our internal logger)